chameleonultragui/lib/gui/page/home.dart

```diff
@@ -317,449 +299,173 @@ class HomePageState extends State<HomePage> {
                                 if (latestCommit.startsWith(fwVersion[1])) {
                                   snackBar = SnackBar(
                                     content: Text(
-                                        'Your Chameleon ${appState.connector.device == ChameleonDevice.ultra ? "Ultra" : "Lite"} firmware is up to date'),
-                                    action: SnackBarAction(
-                                      label: 'Close',
-                                      onPressed: () {},
-                                    ),
+                                        'Your ${appState.connector.device.name} firmware is up to date'),
+                                    showCloseIcon: true,
                                   );
 
-                                  ScaffoldMessenger.of(context)
-                                      .showSnackBar(snackBar);
+                                  scaffoldMessenger.showSnackBar(snackBar);
                                 } else {
+                                  var message = 'Downloading and preparing new ${appState.connector.device.name} firmware...';
+                                  if (appState.onWeb) {
+                                    message = 'Your ${appState.connector.device.name} firmware is out of date! Automatic updates are not (yet) supported on web, download manually and then update by clicking on the Settings icon';
```

Comment:
Support them

Reply:
That todo is for @GameTec-live, he basically needs to tell me which proxy to use

Reply:
https://corsproxy.org/

Reply:
Do you know who is behind that? Preferably we should also think about privacy, we don't want to expose all web users of this app to an unknown third party.

My preference would be to keep the proxy in-house, should be quite easy with cloudflare/aws/netlify etc.

Or maybe we could create a github action that runs every night and publishes the latest releases + zip's on a github.io site?

Reply:
ima just create: corsproxy.gametec-live.com
and host it on my homeserver, as in house as its gonna get lol

Reply:
or if @Foxushka would share the chameleon.run domain via cloudflare withit i could setup cors.chameleon.run
while chameleon.run is webapp...

Reply:
If you are using cloudflare already its also possible to run the proxy as a cloudflare worker which supports caching. Not sure about costs though

Reply:
dns sht, cors.gametec-live.com is cloudflare worker cors proxy.

Reply:
@pimlie you didn't saw our proxy wars. I hope you will do correct decision on proxy

Reply:
![image](https://github.com/GameTec-live/ChameleonUltraGUI/assets/135865149/a0056b66-7f2d-42c3-86b8-34f35e8f65d6)